README.md

```diff
@@ -23,6 +23,14 @@ placing a link of the following kind in the markup:
     print "<a href='phpstorm://open?file=$file&line=$line'>Open with PhpStorm</a>";
     ?>
 
+Added in ability to path map to local files from docker/devilbox.
+
+Edit phpstorm-url-handler file, find the following lines:
+find="/shared/httpd/"
+replace="/home/michael/public_html/";
```

Comment:
Both of these lines show on the single line after README is rendered. This is how you planned this to be?